src/component/ButtonLike/ButtonLike.js

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+import {FaHeart} from 'react-icons/fa';
+import styled from 'styled-components';
+
+export default function ButtonLike() {
+  const [isLiked, setIsLiked] = useState(false);
+
+  function toggleLike() {
+    setIsLiked(!isLiked);
+  }
+
+  return (
+    <StyledButton onClick={toggleLike} aria-label="like">
+      <StyledHeartIcon $isLiked={isLiked} />
```

Comment:
Nice solution, but  take care of your props naming, please remove the `$` 😉
```suggestion
      <StyledHeartIcon isLiked={isLiked} />
```